cmd/trace-agent/run.go

```diff
@@ -251,45 +251,51 @@ func Run(ctx context.Context) {
 type corelogger struct{}
 
 // Trace implements Logger.
-func (corelogger) Trace(v ...interface{}) { log.Trace(v...) }
+func (corelogger) Trace(v ...interface{}) { log.TraceStackDepth(3, v...) }
```

Comment:
Any particular reason `trace-agent` needs a layer on top of `pkg/util/log` ?
It seems like a lot of duplicated code...

Reply:
Good point. It looks like `pkg/trace` has work to become an independent module, so the `corelogger` was introduced in this [PR](https://github.com/DataDog/datadog-agent/pull/11177) to support that. My PR is just aiming to fix some bugs on top of it, so I'm unsure of the details beyond that, unfortunately :(